automatic/PDFCreator/update.ps1

```diff
@@ -0,0 +1,24 @@
+import-module au
+
+$url = 'http://download.pdfforge.org/download/pdfcreator/PDFCreator-stable?download'
+
+function global:au_SearchReplace {
+    @{
+        'tools\ChocolateyInstall.ps1' = @{
+            "(^[$]checksum\s*=\s*)('.*')"   = "`$1'$($Latest.Checksum32)'"
+        }
+     }
+}
+
+function global:au_GetLatest {
+    Write-Host $url
+
+    $temp_file = $env:TEMP + '\PDFCreator.exe'
+    Invoke-WebRequest $url -OutFile $temp_file
+
+    $version = (Get-Command $temp_file).Version
```

Comment:
I wouldnt add 0 just like that. What happens when they tomorrow release 2.5.0.1. You would get invalid version. I think u should use site version.

2.5 is not equal to 2.5.0 BTW.